server/routes/notifications.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+
+const { 
+    createNotification,
+    notificationList,
+    updateReadStatus,
+} = require("../controllers/notifications");
+
+router.route("/create").post(protect, createNotification);
+router.route("/all").get(protect, notificationList);
+router.route("/read").patch(protect, updateReadStatus);
```

Comment:
If the base url is `localhost/notications`, then these endpoints should be:
- `router.route('/').post(...)`
- `router.route('/').get(...)`
- `router.route('/:notificationId').patch(...)`

Reply:
@ezeilo-su  sorry, I did not meant to say this. I was just trying to explain to use local host url to test it.

Reply:
@ezeilo-su  ready for a second review!

Reply:
done